src/OrchardCore/OrchardCore.Notifications.Core/Models/NotificationOptions.cs

```diff
@@ -11,4 +11,10 @@ public class NotificationOptions
     /// Whether or not to disable HtmlBody Sanitizer.
     /// </summary>
     public bool DisableNotificationHtmlBodySanitizer { get; set; }
+
+    /// <summary>
+    /// How may seconds should the top-unread notification be cache for.
+    /// Null value will indicate no cache.
+    /// </summary>
+    public int? CacheDurationInSeconds { get; set; } = 3600;
```

Comment:
`0` should be the value to put to disable the cache, since this is also what is used in the code. Meaning remove the nullability and adapt the doc.

Reply:
okay. done.